zaza/openstack/charm_tests/cinder_netapp/tests.py

```diff
@@ -55,23 +65,28 @@ def test_cinder_config(self):
             expected_contents,
             timeout=2)
 
+    def _create_volumes(self, n):
+        """Create N volumes."""
+        for _ in range(n):
+            name = "zaza{}".format(uuid.uuid1().fields[0])
+            vol = self.cinder_client.volumes.create(
+                name=name,
+                size='1')
+            vol.reset_state('available')
+            self.zaza_volumes.append(vol)
+
     def test_create_volume(self):
-        """Test creating a volume with basic configuration."""
-        test_vol_name = "zaza{}".format(uuid.uuid1().fields[0])
-        vol_new = self.cinder_client.volumes.create(
-            name=test_vol_name,
-            size='1')
-        try:
+        """Test creating volumes with basic configuration."""
+        self._create_volumes(2)
+        for vol in self.cinder_client.volumes.list():
+            self.assertTrue(vol)
```

Comment:
It's not `True`; it's just not `None`; shouldn't this assert be for what it actually is, as `True` can't have an `.id` later in the test (on line 85).

Reply:
From what I understand, `assertTrue` is basically `bool(x)`, which helps if the volume class implements the `__bool__` protocol (don't know if it does, just wanted to be cautious).

Reply:
It's not that it *won't* work; I'm absolutely certain it will.  It's that it *says* "assert that vol is True", whereas I think you are actually asserting that `vol` is not None, which is the only alternative to vol not being an object that you can later get an attribute on (`is`).  My issue is that I think code should say what it means: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone

Reply:
Sounds reasonable. I'll change it.